raiden-ts/src/transfers/utils.ts

```diff
@@ -102,7 +106,7 @@ export function raidenSentTransfer(sent: SentTransfer): RaidenSentTransfer {
     token: sent.transfer[1].token,
```

Comment:
nitpick: If we are not limited by the number of lines in the function, would it make sense `sent.transfer[1]` into a variable instead of repeating?